client/src/components/Workflow/InvocationExport/InvocationExport.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+    <span class="workflow-invocation-export-component">
+        <h2>Export worklfow invocation</h2>
```

Comment:
```suggestion
        <h2>Export workflow invocation</h2>
```

Reply:
Thanks Nicola for the catch, but I'm considering dropping this component in favor of the plugin system. I want to discuss it with John first :)